content/influxdb/v2.2/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,81 @@ menu:
 weight: 101
 ---
 
+## v2.2 [TBD]
+
+### Features
+
+#### Replicate data remotely
+
+- Add the option to [replicate data from InfluxDB OSS to InfluxDB Cloud](/influxdb/v2.2/write-data/replication).
+
+This feature helps you to collect data locally from multiple InfluxDB OSS edge deployments, analyze and aggregate the data, and then send the data to one or more buckets on a remote InfluxDB Cloud instance. If InfluxDB OSS loses connectivity to InfluxDB Cloud, the data is sent when network connectivity is restored.
+
+#### Flux updates
+
+- Update to [Flux v0.150.0](/flux/v0.x/release-notes/#v01500-2022-01-19).
+- Option to log Flux queries that are cancelled because of server shutdown.
+
+#### Build maintenance
+
+- Upgrade `protobuf` library.
+- https://github.com/influxdata/influxdb/commit/5a1e375): Build: allow to build on FreeBSD
+
+#### Task metadata
+
+- (https://github.com/influxdata/influxdb/commit/504f0e4): Passing `type=basic` returns task metadata without query text.
```

Comment:
Is this an option the user can select or pass somewhere, or solely behind the scenes?

Reply:
@kelseiv It's in the public API (https://github.com/influxdata/openapi/blob/6d3d0c2267e5fd55d6b832d491e2a14ba4cf2c96/contracts/ref/oss.yml#L11491), though the primary user seems to be the InfluxDB UI.